engines/native/sandbox.go

```diff
@@ -230,7 +254,7 @@ func (s *sandbox) Abort() error {
 		}
 
 		// Remove temporary home folder
-		err := s.workingFolder.Remove()
+		err := os.RemoveAll(s.workingFolder)
```

Comment:
So now you're remove the home directory that the current user has... I think that can have negative side-effects :)
If reusing a home directory, I don't think we can do any cleanup.